regolith/builders/recentcollabsbuilder.py

```diff
@@ -17,8 +20,16 @@
 from regolith.sorters import doc_date_key, ene_date_key, position_key
 from regolith.builders.basebuilder import LatexBuilderBase, latex_safe
 
-LATEX_OPTS = ["-halt-on-error", "-file-line-error"]
+# specifying border and font style
```

Comment:
do we need this?  Doesn't jinja2 just inherit this from the template?

Reply:
actually 48 is the number of months from today to search from (searching for all collaborators who worked with "sbillinge" within 48 months from today). There's a global variable for the lines to skip to the collaborators table in the excel file, `COAUTHOR_TABLE_OFFSET`, declared at the beginning of the code.